gap_quantization/dump_utils.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+import os.path as osp
+import re
+import shutil
+
+import torch
+import torch.nn as nn
+
+#from urllib.parse import urlparse
+
+
+def dump_quant_params(save_dir, convbn):
+    dict_norm = {}
+    for file in os.listdir(save_dir):
+        if file != 'activations_dump' and not re.match('.*cat.json', file) and not re.match('.*txt', file):
+            dict_norm[file] = read_norm(os.path.join(save_dir, file))
+    list_norm = make_list_from_dict(dict_norm, convbn)
+    txt_list = open('param_layer_quant.h', 'w')
```

Comment:
open() requires a corresponding `.close()` call.
Another option is to use `with`:
```
with open('param_layer_quant.h', 'w') as txt file:
    for i in range(0, 26):
        ...
```